### DISCUSSION TITLE
Feature Request: Adhoc Custom Prompt

### DISCUSSION THREAD

**Discussion by krittaprot** [pfrankov/obsidian-local-gpt#47] [Ideas]
Hello,

I've tested the plugin and had a great experience so far, but I do wish it included an option to specify custom ad-hoc prompts (similar to apple intelligence writing tools) when the context menu is triggered. Would you consider adding this feature in a future update?

**Comment by pfrankov** [comment]
Hello there! You can use just plain text with **🪄 General ~~Kenobi~~ help**, like that
<img src="https://private-user-images.githubusercontent.com/584632/291003532-724d4399-cb6c-4531-9f04-a1e5df2e3dad.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3NDc4NzEsIm5iZiI6MTczMDc0NzU3MSwicGF0aCI6Ii81ODQ2MzIvMjkxMDAzNTMyLTcyNGQ0Mzk5LWNiNmMtNDUzMS05ZjA0LWExZTVkZjJlM2RhZC5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwNFQxOTEyNTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMzI4ODJkNDA3NTc4MWVmNzJmYTgzNmMwZTFiN2IxMjRiZTViOGRlYzYyMDUzMWJiNTg1MTc0Y2YzZWVhZTYzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bnp7g8OxjTQzwBqv4cnJ24tYng7JHz4MVk01V2JRqJ0"/>
